scipy/optimize/__init__.py

```diff
@@ -86,6 +86,7 @@
 
    leastsq - Minimize the sum of squares of M equations in N unknowns
    nnls - Linear least-squares problem with non-negativity constraint
+   lsq_linear - Linear least-squares problem with arbitrary bound constraints
```

Comment:
Don't use "bounded-variable"    I would think of, for example, a logit model where the dependent variable has bounds (0, 1)

(I would prefer "bound constraints" dropping the "arbitrary", or "box constraints" which sounds like scipy lingo)

(from the sidelines)